.pre-commit-config.yaml

```diff
@@ -21,6 +21,8 @@ default_stages: [pre-commit, pre-push]
 default_language_version:
   # force all unspecified Python hooks to run python3
   python: python3
+  # force all unspecified Node hooks to run Node.js v22.14.0 LTS
+  node: 22.14.0
```

Comment:
Node.js `v22.14.0` is the LTS version we should be now using.

https://nodejs.org/en

And seen on the pre-commit site we can set the node version as default.

For node they use nodeenv.

https://pre-commit.com/#overriding-language-version

And here shows how nodeenv uses and lists its version numbers without the `v` prefix:

`nodeenv --list`

https://github.com/ekalinin/nodeenv?tab=readme-ov-file#advanced